gpii/node_modules/packagekit/nodepackagekit/nodepackagekit.cc

```diff
@@ -149,45 +179,46 @@ Handle<Value> searchPackage(const Arguments& args) {
  *            Special case:  the "none" filter = installed or available.
  * Returns: Array of packages matching the search criteria.
  */
-Handle<Value> searchFiles (const Arguments& args) {
-  HandleScope scope;
+NAN_METHOD(searchFiles) {
   PkClient *client = NULL;
-  PkResults *results = NULL;
+  PkResults *pkResults = NULL;
   GPtrArray *array;
   PkPackageSack *sack = NULL;
   std::basic_string<gchar> pathName;
   PkBitfield filtersBitField;
   GError *err = NULL;
-  v8::Handle<v8::Object> pac;
-  v8::Handle<v8::Array> result = v8::Array::New();
+  Handle<Object> pac;
+  Handle<Array> result;
 
-  pathName = utf8StringFromValue(args[0]);
+  pathName = utf8StringFromValue (info[0]);
   gchar *pathNames[sizeof(pathName)] = {&pathName[0]};
-  filtersBitField = handleFiltersArgument (args, 1);
+  filtersBitField = handleFiltersArgument (info, 1);
 
   client = pk_client_new();
-  results = pk_client_search_files (client, filtersBitField,
-                                    pathNames, NULL,
-                                    (PkProgressCallback) progressCallback, NULL, &err);
+  pkResults = pk_client_search_files (
+    client, filtersBitField, pathNames, NULL,
+    (PkProgressCallback) progressCallback, NULL, &err
+  );
 
-  sack = pk_results_get_package_sack(results);
-  pk_package_sack_sort(sack, PK_PACKAGE_SACK_SORT_TYPE_NAME);
-  array = pk_package_sack_get_array(sack);
+  sack = pk_results_get_package_sack(pkResults);
+  pk_package_sack_sort (sack, PK_PACKAGE_SACK_SORT_TYPE_NAME);
```

Comment:
A holdover from my GNOME programming days.  The [GNOME style guide](https://developer.gnome.org/programming-guidelines/stable/c-coding-style.html.en) requires spaces after function names.  Is there a gpii c/c++ style guide? ;-)

I'll remove the spaces.